src/VisualStudio/Core/Def/Implementation/Diagnostics/DiagnosticProgressReporter.cs

```diff
@@ -118,9 +105,6 @@ private void ProgressUpdated()
                 // make sure we are not flooding UI. 
```

Comment:
Do all these comment need to be deleted now? This class is no longer responsible for the throttling?